doc/Language/quoting.pod6

```diff
@@ -216,9 +204,9 @@ say "@neighborsB<[]> and I try our best to coexist peacefully."
 Felix Danielle Lucinda and I try our best to coexist peacefully.
 =end code
 
-Often a method call is more appropriate. These are allowed within C<qq>
-quotes as long as they have parentheses after the call. Thus the following
-code will work:
+Alternatively, rather than using C<[]>, you can interpolate the Array by
+following it with a method call with parentheses after the method name. Thus the
+following code will work:
```

Comment:
All this seems strangely ad hoc and does not seem to match the "strangely consistent" nature of Raku. There must be some underlying simple rule we're not getting